Content.Client/Popups/PopupSystem.cs

```diff
@@ -78,13 +79,22 @@ private void PopupMessage(string? message, PopupType type, EntityCoordinates coo
                     _replayRecording.RecordClientMessage(new PopupCoordinatesEvent(message, type, GetNetCoordinates(coordinates)));
             }
 
+            var popupData = new WorldPopupData(message, type, coordinates, entity);
+            if (_aliveWorldLabels.TryGetValue(popupData, out var existingLabel))
+            {
+                existingLabel.TotalTime = 0;
+                existingLabel.Repeats += 1;
+                existingLabel.Text = $"{popupData.Message} x{existingLabel.Repeats}";
```

Comment:
the x represents the number of times repeated, which may change per culture, make loc string